### ISSUE TITLE
Latissimus dorsi rupture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11040]
**Preferred term label:**
Latissimus dorsi rupture

**Synonyms**

**Definition (free text, please give PubMed ID)**
Advanced imaging showing partial or full thickness tear of the muscular or tendinous portion of the latissimus dorsi

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Latissimus dorsi rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added
HP:0020320
A tear of the latissimus dorsi (LD) tendon. TheLD is a broad, flat muscle occupying most of the lower posterior thorax. The muscle's primary function is to move the upper extremity, but it is also considered an accessory muscle of respiration.

**Comment by MickeySegal** [comment]
Does this term only include the tendon?  Do we need a separate term for the muscle?

**Comment by pnrobinson** [comment]
@MickeySegal 
you are right, thanks for pointing this out. I have updated the definition to

A tear of the latissimus dorsi (LD). Injury to the LD can occur at various locations, from the tendon to the muscle belly. The LD is a broad, flat muscle occupying most of the lower posterior thorax. The muscle's primary function is to move the upper extremity, but it is also considered an accessory muscle of respiration.
